api/meta_protocol_proxy/v1alpha/meta_protocol_proxy.proto

```diff
@@ -151,3 +154,10 @@ message Tracing {
   envoy.config.trace.v3.Tracing.Http provider = 9;
 }
 
+message ProtocolConfig {
+  // The codec which encodes and decodes the application protocol.
+  Codec codec = 1;
+  // is open multiplexing
+  bool open_multiplexing = 2;
```

Comment:
I think just multiplexing is clear enough.

Reply:
ok